docs/static_site/src/_config_prod.yml

```diff
@@ -59,6 +59,12 @@ markdown: kramdown
 plugins:
   - jekyll-feed
   - jekyll-seo-tag
+
+# Force include .asf.yaml
+include:
+  - .asf.yaml
+  - .htaccess
```

Comment:
why are we adding this? it was already included in Jekyll build.

Reply:
The log shows html/.htaccess is not included during building jekyll docs. 
https://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/restricted-website-build-master-beta/detail/restricted-website-build-master-beta/5/pipeline/219

Reply:
Probably because we update the `include` in config, which overrides the default config that includes `.htaccess`

Reply:
thanks. that makes sense. can we also add a similar check for .asf.yaml here?
https://github.com/apache/incubator-mxnet/blob/e359bcd65e453d4bc86d3d8e5b1dee3916a2e426/ci/docker/runtime_functions.sh#L1207